spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListener.java

```diff
@@ -89,11 +90,13 @@
  * @since 1.4
  * @see EnableRabbit
  * @see RabbitListenerAnnotationBeanPostProcessor
+ * @see RabbitListeners
  */
 @Target({ ElementType.TYPE, ElementType.METHOD, ElementType.ANNOTATION_TYPE })
 @Retention(RetentionPolicy.RUNTIME)
 @MessageMapping
 @Documented
+@Repeatable(RabbitListeners.class)
```

Comment:
The `@Repeatable` is `@since 1.8`.
How does it compile on Java < 8?
Or have I just forgotten that Java doesn't care about unknown annotations?..